src/components/index.js

```diff
@@ -32,6 +32,7 @@ export { default as OptionSeparator } from './option-separator';
 export { default as Sidebar, SidebarItem } from './sidebar';
 export { default as SocialPrefixedInput } from './social-prefixed-input';
 export { default as StackedAvatar } from './stacked-avatar';
+export { default as AvatarCard } from './avatar-card';
```

Comment:
👍 the line what I usually forget.

Reply:
to make sure we've done this, we should import the component from the index file when we want to render them on the page as an example. It would enforce us the expose it in the index file. Just an idea. cc @gazdagergo